demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs

```diff
@@ -145,8 +145,43 @@ @@                   @@@
     private static void DrawMac()
     {
         Console.WriteLine("Hello, Mac...");
-        Console.WriteLine();
 
-        // TODO: Something pretty here :)
+        const string Apple = @"
+                                   ,++
+                               @@@@@@+
+                             @@@@@@@@
+                           @@@@@@@@@:
+                          @@@@@@@@@.
+                         :@@@@@@@@
+                         @@@@@@;
+                         `
+         ,@@@@@@@@@@@:        @@@@@@@@@@@@@:
+      :@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@:
+    :@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+   @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@'
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@`
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+   @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+    @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+     @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+      @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+       @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+         @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+          :@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@;
+             @@@@@@@;           ;@@@@@@@";
+
+        Console.ForegroundColor = ConsoleColor.White;
```

Comment:
Consider setting both ForegroundColor and BackgroundColor here to ConsoleColor.Red or White or something similar.